CONTRIBUTING.md

```diff
@@ -299,9 +299,26 @@ Alternatively, or complimentary to that, you can use `tox` -- there is a `tox.in
 file which sets up a few virtual environments for testing locally, which you can
 later reuse like any other regular virtualenv for troubleshooting.
 Additionally, [tools/testing/test_README_in_docker](tools/testing/test_README_in_docker) script can
-be used to establish a clean docker environment (based on any NeuroDebian-supported
+be used to establish a clean docker environment (based on any NtesteuroDebian-supported
```

Comment:
Unless this is the new bitcoin wallet for NeuroDebian, I guess it should be removed again ;-)

Reply:
Typo was fixed in maint already... I will then merge maint and this into master locally and push